modules/eco-self-node-remediation-operator-about.adoc

```diff
@@ -67,15 +73,13 @@ The `SelfNodeRemediationTemplate` CR resembles the following YAML file:
 apiVersion: self-node-remediation.medik8s.io/v1alpha1
 kind: SelfNodeRemediationTemplate
 metadata:
-creationTimestamp: "2022-03-02T08:02:40Z"
-generation: 1
-name: self-node-remediation-default-template
-namespace: openshift-operators
-resourceVersion: "596469"
-uid: 5d29e437-c485-48fa-ba9e-0354649afd31
-spec:
-template:
+  creationTimestamp: "2022-03-02T08:02:40Z"
+  name: self-node-remediation-<remediation_object>-deletion-template <1>
+  namespace: openshift-operators
 spec:
-remediationStrategy: NodeDeletion <1>
+  template:
+    spec:
+      remediationStrategy: <remediation_strategy>  <2>
 ----
-<1> Specifies the remediation strategy. The default remediation strategy is `NodeDeletion`.
+<1> Specifies the type of remediation template based on the remediation strategy. Replace `<remediation_object>` either `resource` or `node`, for example, `self-node-remediation-resource-deletion-template`.
```

Comment:
You could update "Replace `<remediation_object>` either" to "Replace `<remediation_object>` with either".

Reply:
Done; sorry for the silly typos